data/json/recipes/weapon/bashing.json

```diff
@@ -189,6 +189,33 @@
     "tools": [ [ [ "drift", -1 ] ] ],
     "components": [ [ [ "stick_long", 1 ] ], [ [ "fur", 2 ], [ "leather", 2 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "survivor_polehammer",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_BASHING",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": "30 m",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 2 }, { "id": "HAMMER", "level": 1 } ],
+    "proficiencies": [ { "proficiency": "prof_carving", "time_multiplier": 1.5, "skill_penalty": 0.15 } ],
+    "components": [ [ [ "spear_shaft", 1 ] ], [ [ "hammer", 1 ], [ "survivor_polehammer_makeshift", 1 ] ], [ [ "nails", 3, "LIST" ] ] ]
+  },
```

Comment:
Then how do nails make that more stable? I don't think this justifies durable_melee particularly, and FRAGILE_MELEE may be justified for the lesser version.

Reply:
Its not nails that make the most difference, but they are often used in mounting hammerheads and axeheads. The real difference comes from propper spearshaft, the same that is used in crafting other, weapon grade polearms. Best possible wooden handle + baisicaly indestructible but small hammerhead = durable weapon

Reply:
My understanding is that mounting a head with nails looks like this:
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/42699974/3e110a3d-9a5f-4cce-b9db-5946b998ab1c)
Which is not something you can do with a pre-fabricated hammer.

Reply:
What I had in mind with nails was using them as wedges. I know most professional wedges for hammer and axe heads are small metal trianges, in CDDA that would probably translate to... scrap metal? Or small metal sheet. And that would be more confusing

Reply:
Small metal sheets + nuts & bolts instead of nails should be used for mounting then

Reply:
I dont understand how nuts and bolts can be used as wedges